wordpress-mobile/WordPress-iOS
# Remove Dangerfile, gem and CircleCI job

Danger has now been replaced with its self-hosted equivalent, [Peril](https://github.com/danger/peril).

All existing rules from the `Dangerfile` have been added to our new Peril settings repo: https://github.com/wordpress-mobile/peril-settings. This will allow us to share rules across projects and platforms.

You can see an example of Peril failing here: https://github.com/wordpress-mobile/WordPress-iOS/pull/10931#issuecomment-464795861.

You can see Peril running on a forked PR here: https://github.com/wordpress-mobile/WordPress-iOS/pull/11102

To test:

- See that there is a new Peril status on this PR, and on the status page: https://github.com/wordpress-mobile/WordPress-iOS/pull/11075/checks?check_run_id=65070672.

Update release notes:
